test/parallel/test-worker-message-port-wasm-module.js

```diff
@@ -0,0 +1,19 @@
+// Flags: --experimental-worker
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const fixtures = require('../common/fixtures');
+
+const { Worker } = require('worker_threads');
+const wasmModule = new WebAssembly.Module(fixtures.readSync('test.wasm'));
+
+const worker = new Worker(`
+const { parentPort } = require('worker_threads');
+parentPort.once('message', ({ wasmModule }) => {
+  const instance = new WebAssembly.Instance(wasmModule);
+  parentPort.postMessage(instance.exports.addTwo(10, 20));
```

Comment:
can we also verify this with a shared memory export from the wasm

Reply:
@devsnek You mean like the fixup commit in https://github.com/nodejs/node/pull/25307?

Reply:
@addaleax like the first test, not the second

Reply:
Uh… I’m not sure what you mean, then.

Reply:
post a wasm module with shared through a message channel and verify that its the correct memory and modifying it works and such.

Reply:
Afaict that’s not possible without #25307, and I wouldn’t block either PR on that…